R/in-line.R

```diff
@@ -191,19 +191,19 @@
 d_max <- function(x, low, high, scale = 1, missing = NA_real_, use_data = FALSE) {
   low  <- check_args(low,  x, use_data, fn = "d_max")
   high <- check_args(high, x, use_data, fn = "d_max", type = "high")
-  check_scale(scale)
+  check_scale(scale, arg = "scale", call = rlang::current_call())
```

Comment:
I find it easier if the checking function has this signature:

```r
check_scale <- function(x, arg, call = rlang::caller_env()) {
```

because then you don't need to supply a `call` if it is used at the top of a function. This line would be

```r
  check_scale(scale, arg = "scale")
```